how-to-use-azureml/automated-machine-learning/model-explanation/auto-ml-model-explanation.yml

```diff
@@ -0,0 +1,9 @@
+name: auto-ml-model-explanation
```

Comment:
This file is not in AzureMlCli and should not be pushed to GitHub.
It is generated as part of running the associated notebook.